Assets/Scripts/Inventory/InventoryItem_Equippable.cs

```diff
@@ -16,6 +18,36 @@ private void Reset()
 
     public override void ContextAction()
     {
-        Logger.log("Equip " + name);
+        Logger.log("Toggle Equip " + name);
+        if (equipped)
+        {
+            Unequip();
+            equipped = false;
+            this.contextActionName = "Equip";
```

Comment:
This concept of ""contextActionName" is inelegant. Why not let the items provide multiple context actions without hardcoded names in the code like this?
Have different methods for different context actions, like Use, Consume, Equip, etc.